### PR TITLE
Avoid setting Evict workloadUpdates strategy

### PR DESCRIPTION
**Pull request by tiraboschi**
Evict workloadUpdates strategy can be disruptive
for existing workloads,
avoid setting it by default and let the user
eventually opt-in.

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2008900

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:
```release-note
Avoid setting Evict workloadUpdates strategy
```

### CODE REVIEW

**Comment by tiraboschi** [comment]
/hold
until we get a final decision on this

**Comment by tiraboschi** [comment]
hco-e2e-image-index-aws passed
/override ci/prow/hco-e2e-image-index-azure
/override ci/prow/hco-e2e-image-index-gcp

**Comment by tiraboschi** [comment]
hco-e2e-kv-smoke-azure passed
/override ci/prow/hco-e2e-kv-smoke-gcp

**Comment by tiraboschi** [comment]
okd-hco-e2e-upgrade-index-gcp passed
/override ci/prow/okd-hco-e2e-upgrade-index-aws

**Comment by tiraboschi** [comment]
okd-hco-e2e-image-index-aws passed
/override ci/prow/okd-hco-e2e-image-index-gcp

**Comment by tiraboschi** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=2008900 got marked as blocker+
/unhold

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**
